src/data/faq_de.json

```diff
@@ -65,11 +194,28 @@
                         "indexed": true,
                         "accordion": [
                             {
-                                "title": "Hier bereiten wir Antworten zu Fragen vor, welche Möglichkeiten es nach Ende der Entwicklungsarbeiten an der CWA künftig noch gibt.",
-                                "anchor": "ramp_down_alternatives",
+                                "title": "Was passiert mit den CovPass-Apps? Werden sie auch eingestellt? Wenn ja, wann?",
+                                "anchor": "ramp_down_covpass",
+                                "active": false,
+                                "textblock": [
+                                    "Die CovPass-Apps werden in Übereinstimmung mit der Geltungsdauer der europarechtlichen Rechtsgrundlagen weiter betrieben. Die hierfür maßgebliche Verordnung (EU) 2021/953 gilt derzeit bis zum 30. Juni 2023."

```

Comment:
A reference to the CovPass FAQs would be well-placed here, I think.

Reply:
But that's definitely only a nice to have, but not a must-have.

Reply:
We already tried to build a bridge from CWA to other apps and their planned lifecycle. All those questions and remarks got removed from the FAQ proposal, with the comment that it belongs to general pandemic management, but not related to the "end of CWA development" 

So, for now, we do not refer to other apps (not even to CovPass), as there is not even a legal basis to utilize one of those.

Reply:
@GisoSchroederSAP 

> So, for now, we do not refer to other apps (not even to CovPass), as there is not even a legal basis to utilize one of those.

Does that mean you will remove the entry #ramp_down_covpass from the proposed FAQ changes?